core/src/main/java/org/akaza/openclinica/service/user/LdapUserService.java

```diff
@@ -75,7 +75,7 @@
 
     private SpringSecurityLdapTemplate ldapTemplate;
 
-    @PostConstruct // Eclipse warning here is an Eclipse bug, not an issue with the code

```

Comment:
Why is this annotation removed? it seems to me that if it isn't called the ldapTemplate stays null and may result in the reported NullPointerException. Am I wrong?

Reply:
In both this case and the below case, I got the following exception:

> [ERROR] \Users\Lenovo\git\LibreClinica\core\src\main\java\org\akaza\openclinica\service\user\LdapUserService.java:[15,23] error: package javax.annotation does not exist
> [ERROR] \Users\Lenovo\git\LibreClinica\core\src\main\java\org\akaza\openclinica\service\user\LdapUserService.java:[78,5] error: cannot find symbol
> [INFO] 2 errors 

The error appeared on both running mvn from the command line in Linux and running it through the Eclipse IDE.

Reply:
Ok, now I get the point. You're working with a JDK-11+ environment and the old packages aren't available anymore. We do want to migrate from Java-8 to newer versions but currently had no time to update LibreClinica accordingly. Can you run Java-8 in your development environment? If not we can try to priories the migration.

Reply:
Aha, I did indeed compile and build with the default JDK included in Eclipse, which was 11+.  Changing back to Java 8 and reintroducing these lines, and it compiles fine.  I'll close the other ticket as well.